packages/console/src/components/RoleEntitiesTransfer/components/SourceEntitiesBox/index.tsx

```diff
@@ -48,17 +48,10 @@ function SourceEntitiesBox<T extends User | Application>({
   };
 
   const { data, error } = useSWR<[Props<T>['selectedEntities'], number], RequestError>(
-    roleType === RoleType.User
-      ? buildUrl('api/users', {
-          excludeRoleId: roleId,
-          ...commonSearchParams,
-        })
-      : buildUrl(`api/applications`, {
-          excludeRoleId: roleId,
-          ...commonSearchParams,
-          'search.type': ApplicationType.MachineToMachine,
-          'mode.type': 'exact',
-        })
+    buildUrl(`api/${roleType === RoleType.User ? 'users' : 'applications'}`, {
+      excludeRoleId: roleId,
+      ...commonSearchParams,
+    })
```

Comment:
don't need to pass `types=` for applications here?

Reply:
Updated, built the search parameter in the parent component.